src/main/java/tech/andreagreco/FileReaderTest.java

```diff
@@ -0,0 +1,20 @@
+package tech.andreagreco;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class FileReaderTest {
+    private void readFile(String fileName) throws IOException {
+        FileInputStream in = null;
+        FileOutputStream out = null;
+
+        in = new FileInputStream("input.txt");
+        out = new FileOutputStream("output.txt");
```

Comment:
Could you please merge declaration and assignment into 1 line to improve readability?